internal/cache/controller_obj_cache.go

```diff
@@ -1931,12 +1931,12 @@ func (c *AviObjCache) PopulateL4PolicySetToCache(client *clients.AviClient, clou
 
 func (c *AviObjCache) AviPopulateAllStringGroups(client *clients.AviClient, cloud string, StringGroupData *[]AviStringGroupCache, nextPage ...NextPage) (*[]AviStringGroupCache, int, error) {
 	var uri string
-	akoUser := lib.AKOUser
 
 	if len(nextPage) == 1 {
 		uri = nextPage[0].NextURI
 	} else {
-		uri = "/api/stringgroup/?" + "&include_name=true&created_by=" + akoUser
+		//Fetching container specific StringGroups
+		uri = "/api/stringgroup/?" + "&include_name=true&name.contains=" + lib.GetAKOUser()
```

Comment:
I probably missed pushing one commit here which adds a '-' at the end which will help resolve this issue, however, it will again overlap for 
**`ako-gw-cluster-`** and **`ako-gw-cluster-1-`** cases

Reply:
@akshayhavile for gateway api call will be 
`/api/stringgroup/?include_name=true&name.contains=ako-gw-cluster-pod-namespace` so above case is handled for atleast gateways